### DISCUSSION TITLE
go编译后端时，遇到backend/init/router/router.go:73:29: undefined: rou.RouterGroupApp

### DISCUSSION THREAD

**Discussion by littlestone0806** [1Panel-dev/1Panel#7041] [Q&A]
编译架构：loong64
完整报错信息如下：
loongson@loongson-gd:~/1Panel-1.10.20-lts$ goreleaser build --clean --single-target --snapshot --timeout 540m0s
  • skipping validate...
  • cleaning distribution directory
  • loading environment variables
  • getting and validating git state
    • accepting to run without a git repository because this is a snapshot
    • git state                                      commit=none branch=none current_tag=v0.0.0 previous_tag=<unknown> dirty=false
    • pipe skipped                                   reason=disabled during snapshot mode
  • parsing tag
  • setting defaults
  • partial
  • snapshotting
    • building snapshot...                           version=0.0.1-next
  • running before hooks
    • running                                        hook=chmod +x ./script.sh
    • running                                        hook=./script.sh
    • running                                        hook=sed -i 's@ORIGINAL_VERSION=.*@ORIGINAL_VERSION=v0.0.1-next@g' 1pctl
    • running                                        hook=go mod tidy
    • took: 31s
  • ensuring distribution directory
  • setting up metadata
  • writing release metadata
  • loading go mod information
  • build prerequisites
  • building binaries
    • partial build                                  match=target=linux_loong64
    • building                                       binary=dist/1Panel_linux_loong64/1panel
    • took: 5m10s
  ⨯ build failed after 5m41s                         error=failed to build for linux_loong64: exit status 1: # github.com/1Panel-dev/1Panel/backend/init/router
backend/init/router/router.go:73:29: undefined: rou.RouterGroupApp

**Comment by wojiushixiaobai** [comment]
- https://github.com/wojiushixiaobai/1Panel-installer/issues/2
